docs/user-guide/configuration/annotations.md

```diff
@@ -45,7 +45,7 @@ The following annotations are supported by plugins maintained by Screwdriver.cd.
 | screwdriver.cd/collapseBuilds | `true` / `false` | Set it to `true` to collapse all `BLOCKED` builds of the job to the latest build.<br>If you want to apply it to the whole pipeline, set it under `shared`. Default behavior depends on cluster configuration, please check with your cluster admin for that.  |
 | screwdriver.cd/cpu | `MICRO` / `LOW` / `HIGH` / `TURBO` | When using a `k8s` executor, this will allow the user to choose between 0.5 (`MICRO`), 2 (`LOW`), 6 (`HIGH`) and 12 (`TURBO`) CPU resources. Default is `LOW`.<br>   When using a `k8s-vm` executor, this will allow the user to choose between 1 (`MICRO`), 2 (`LOW`), 6 (`HIGH`), and 12 (`TURBO`) CPU resources. Default is `LOW`. |
 | screwdriver.cd/disk | `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 20 GB (`LOW`) and 50 GB (`HIGH`) disk space. Default is `LOW`. |
-| screwdriver.cd/diskSpeed | `HIGH` / `TURBO` | When using a `k8s-vm` executor, this will allow the user to choose between machines with different disk speed. Default is no preference for the disk speed. |
+| (DEPRECATED) screwdriver.cd/diskSpeed | `TURBO` | When using a `k8s-vm` executor, this will allow the user to choose between machines with different disk speed. Default is `TURBO` which is the only disk speed SDv4 now provides (as of 08/30/2019). |
```

Comment:
let's leave this such that other cluster admins (non ouroath folks) might need it.

Can you change it to say `Ask your cluster admin`